lib/sycamore/sycamore/tests/unit/transforms/test_graph_extractor.py

```diff
@@ -82,6 +83,10 @@ def __init__(self):
             super().__init__(model_name="mock_model")
 
         def generate(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None):
+            pass
+
+        async def generate_future(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None):
+            await asyncio.sleep(1)
```

Comment:
remove the sleep, I assume it was to help verify that it was waiting, but shouldn't be needed now and slows down tests.

Reply:
fixed